samples/image-metadata/src/components/AppView/InstallationContent.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Heading, Paragraph, TextField } from '@contentful/forma-36-react-components';
+import { styles } from './styles';
+
+export function InstallationContent({
+  allContentTypesIds,
+  contentTypeId,
+  contentTypeName,
+  setContentTypeName
+}) {
+  const validationMessage = allContentTypesIds.includes(contentTypeId)
+    ? `A content type with id "${contentTypeId}" already exists. Try a different name for the demo content type.`
```

Comment:
```suggestion
    ? `A content type with ID "${contentTypeId}" already exists. Try a different name for the demo content type.`
```

Also, another instance of the word 'demo'.

Reply:
Done